x/clp/types/params_test.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ParamKeyTable(t *testing.T) {
+	_ = ParamKeyTable()
```

Comment:
Can be removed unless it is going to be filled in with a meaningful test. The same is true in other places.